packages/api-server/src/methods/modules/eth.ts

```diff
@@ -1066,6 +1067,7 @@ export class Eth {
         return await Promise.all(
           logs.map(async (log) => {
             const ethTxHash =
+              log.eth_tx_hash ||
```

Comment:
Is `Promise.all` still necessary?

Reply:
Actually, it's not necessary. I will create another refactoring PR to include this suggestion.